chia/protocols/shared_protocol.py

```diff
@@ -49,3 +49,10 @@ class Handshake(Streamable):
     (uint16(Capability.RATE_LIMITS_V2.value), "1"),
     (uint16(Capability.NONE_RESPONSE.value), "1"),
 ]
+
+# capabilities to send to nodes prior to 1.7
```

Comment:
I believe this is now version 1.6.2

Reply:
thanks, leftover comment from before the change